pom.xml

```diff
@@ -552,6 +552,11 @@
                 <enabled>false</enabled>
             </snapshots>
         </repository>
+        <repository>
+            <id>wso2</id>
+            <name>Repository</name>
+            <url>http://dist.wso2.org/maven2</url>
```

Comment:
Hi @shivamriky,

Can you change this url to use `https` instead?

Reply:
I went ahead and made this change.

Reply:
Thank you. I didn't see the message earlier.